### ISSUE TITLE
Disable color

### ISSUE THREAD

**Issue by tstreamDOTh** [CoenraadS/Bracket-Pair-Colorizer-2#153]
I want to have the guiding lines but not bracket coloring. Is that possible?

**Comment by that70schris** [comment]
if you're ok with not having the guidelines be colored, you could make all the colors the same
`bracket-pair-colorizer-2.colors`

**Comment by Xsmael** [comment]
@that70schris
> if you're ok with not having the guidelines be colored, you could make all the colors the same
> bracket-pair-colorizer-2.colors

yes, but it would be more awesome if we can just disable it. because if we do it by the colors, we have to do it again everytime we change the theme, and for users of the peacock extension it will be more work also than neccessary.


@tstreamDOTh  i think **"Disable color for brackets only"** would be a better title

**Comment by that70schris** [comment]
I agree, however it's my understanding that `bracket-pair-colorizer-2.colors` is not something that themes generally modify, and even if they did, user settings should still take precedence.  So, although it's still a workaround, at least you shouldn't have to redo it every time you change themes and should still work with the Peacock extension.

Does this sound right or am I missing something?

**Comment by Xsmael** [comment]
@that70schris Yeah, makes sense.

Sorry for this statement: 
>  and for users of the peacock extension it will be more work also than neccessary
I just had started using the extension that day, (for some reasons ^^') I assumed it changes syntax highlighting too...
 
However
> I agree, however it's my understanding that bracket-pair-colorizer-2.colors is not something that themes generally modify, and even if they did, user settings should still take precedence

That's right, the theme will not change brackets color explicitly, but it will be the theme's default color for text.

Let's say am using a dark theme and my text is white on dark bg, so i set all `bracket-pair-colorizer-2.colors` to white so it feels normal; if i switch to a light theme where text is black on white, then all braces won't be visible unless i change `bracket-pair-colorizer-2.colors` .
That's kind of extreme, but it shows what am trying to explain.

**Comment by adriencarbonaro** [comment]
:+1:  I really love the extension but the ability to simply disable colors would be great. I personally want to use the vertical line and the gutter display of brackets without the colors.

Thanks !

**Comment by enkia** [comment]
I too would love the "Disable color for brackets only" setting.

**Comment by inzider** [comment]
Same here, I would use it if my IDE would not look like a rainbow. Guidelines are nice doh.

**Comment by ronakg** [comment]
Any workarounds for this? How to disable brackets and just keep the scope lines?
